packages/blade/src/components/Button/BaseButton/StyledBaseButton.native.tsx

```diff
@@ -75,13 +88,23 @@ const _StyledBaseButton: React.ForwardRefRenderFunction<TextInput, StyledBaseBut
     };
   });
 
+  const handleOnPress = (event: GestureResponderEvent): void => {
+    if (href) {
+      void openURL(href);
```

Comment:
it should return after this right? do we intend to execute onclick and href both?

Reply:
Yes. People can have their analytics events in `onClick`. Just keeping this close to vanilla JS HTML behaviour. This is also similar in Link button